src/else.js

```diff
@@ -24,7 +34,7 @@ export class Else extends IfCore {
     if (!previous || !previous.au.if) {
       throw new Error("Can't find matching If for Else custom attribute.");
     }
-    let ifVm = previous.au.if.viewModel;
-    ifVm.else = this;
+    this.ifVm = previous.au.if.viewModel;
```

Comment:
The equivalent link to Else viewmodel in If is simply called `else`. For consistency, can we rename `ifVm` to `if` -- or rename `else` to `elseVm`?

Reply:
+1 for elseVm for better understanding